src/components/product/IFXProductCreateEdit.vue

```diff
@@ -149,9 +155,21 @@ export default {
             </IFXItemSelectList>
           </v-col>
         </v-row>
-        <v-row>
-          <v-col>
-            <IFXButton btnType="submit" :disabled="!isSubmittable" @action="submit" />
+        <v-row justify="end">
+          <v-col class="flex flex-grow-1 flex-shrink-0">
+          </v-col>
+          <v-col sm="4">
+            <v-row justify="end" nowrap>
+              <v-col>
+                &nbsp;
+              </v-col>
+              <v-col>
+                <IFXButton btnType="reset" btnText="Reset" @action="init()" />
+              </v-col>
+              <v-col>
+                <IFXButton btnType="submit" :disabled="!isSubmittable" @action="submit" />
+              </v-col>
+            </v-row>
```

Comment:
Is it worth making this a component or make it part of a form component?

Reply:
It's probably best to leave IFXButton as it is.  I think it's used outside of forms at times.